charts/gha-runner-scale-set/values.yaml

```diff
@@ -103,16 +103,12 @@ template:
   ##       image: ghcr.io/actions/actions-runner:latest
   ##       env:
   ##         - name: DOCKER_HOST
-  ##           value: tcp://localhost:2376
-  ##         - name: DOCKER_TLS_VERIFY
-  ##           value: "1"
-  ##         - name: DOCKER_CERT_PATH
-  ##           value: /certs/client
+  ##           value: unix:///run/docker/docker.sock
```

Comment:
The changes introduced on lines 96 - 102 of `charts/gha-runner-scale-set/templates/_helpers.tpl` need to be included here in the comments to stay consistent